defaults/main.yml

```diff
@@ -1,162 +1,117 @@
-###   ( https://github.com/benoitc/gunicorn/issues/797 )
-### in both nginx and gunicorn config files
-archivematica_src_am_dashboard_bind: "127.0.0.1:8002"
-### in nginx config file
+# Reset data options
+archivematica_src_reset_mcpdb: "false"           # Reset AM MCP database
+archivematica_src_reset_shareddir: "false"       # Reset AM shared directory
+archivematica_src_reset_es: "false"              # Reset AM Elasticsearch indices
+archivematica_src_reset_am_all: "false"          # Reset AM Elasticsearch indices, shared directory, MCP database (overrides previous settings)
+archivematica_src_reset_ss_db: "false"           # Reset the SS database
+
+# Database options
+archivematica_src_am_migrate_from_v1_4: "false"  # Migrate AM database from v1.4 to v1.5
+archivematica_src_ss_migrate_from_v0_7: "false"  # Migrate AM database from v1.4 to v1.5
+archivematica_src_am_create_db: "true"           # Create database (uses root's credentials in ~/.my.cnf)
+archivematica_src_am_create_dbuser: "true"       # Create user database (uses root's credentials in ~/.my.cnf)
+archivematica_src_am_wait_local_mysql: "true"    # MCPServer upstart service to wait until MySQL is locally available
+
+# Please note that a playbook using `archivematica_src_am_migrate_from_v1_4` or `archivematica_src_ss_migrate_from_v0_7`  is not idempotent, and it will throw an error if run twice. The two migrate related variables should be removed after a successful migration (an alternative could be to specify the `archivematica_src_am_migrate_from_v1_4` and `archivematica_src_ss_migrate_from_v0_7` at run time using `--extra-vars` instead of putting the variables in the playbook).
+
+
+#
+# Environment strings
+#
+
+# Archivematica Dashboard
+# This list only includes mandatory strings.
+archivematica_src_am_dashboard_environment:
+  DJANGO_SETTINGS_MODULE: "settings.production"
+  ARCHIVEMATICA_DASHBOARD_DASHBOARD_DJANGO_ALLOWED_HOSTS: "*"
+  ARCHIVEMATICA_DASHBOARD_DASHBOARD_DJANGO_SECRET_KEY: "CHANGE_ME_WITH_A_SECRET_KEY"
+  # Not mandatory:
+  ARCHIVEMATICA_DASHBOARD_DASHBOARD_ELASTICSEARCH_SERVER: "127.0.0.1:9200"  # Needed by pipeline-es.yml
+  AM_GUNICORN_BIND: "127.0.0.1:8001"
+
+# Archivematica MCPServer
+# This list only includes mandatory strings.
+archivematica_src_am_mcpserver_environment:
+  DJANGO_SETTINGS_MODULE: "settings.common"
+
+# Archivematica MCPClient
+# This list only includes mandatory strings.
+archivematica_src_am_mcpclient_environment:
+  DJANGO_SETTINGS_MODULE: "settings.common"
+
+# Archivematica Storage Service
+# This list only includes mandatory strings.
+archivematica_src_ss_environment:
+  DJANGO_SETTINGS_MODULE: "storage_service.settings.production"
+  DJANGO_ALLOWED_HOSTS: "*"
+  DJANGO_SECRET_KEY: "CHANGE_ME_WITH_A_SECRET_KEY"
+  SS_DB_HOST: ""
+  SS_DB_NAME: "/var/archivematica/storage-service/storage.db"  # Needed by ss-db.yml
+  SS_DB_PASSWORD: ""
+  SS_DB_USER: ""
+  # Not mandatory:
+  SS_GUNICORN_BIND: "127.0.0.1:8002"
+
```

Comment:
I notice AM / SS back end ports are swapped from what they were previously 8001 <-> 8002. Makes sense to me I guess, to have the AM port 'first'. Although caused some confusion as I sometimes kill the gunicorn service and replace it with a runserver for easier debugging! Is there anywhere these ports are documented that will need to be changed?